powerfulseal/policy/scenario.py

```diff
@@ -68,6 +72,43 @@ def execute(self):
         self.cleanup()
         return True
 
+    def retry(self, step_retry, step_action, action_method):
+        retry_sleep = 30
+        ret = False
+        if "retriesTimeout" in step_retry.keys():
+            if "timeout" in step_retry['retriesTimeout'].keys():
+                final_timeout = step_retry['retriesTimeout']['timeout']
+            else:
+                final_timeout = 60
```

Comment:
In other places, we used this pattern instead - you might find it a little bit easier to read:

```py
 final_timeout = step_retry.get('retriesTimeout', {}_).get('timeout', 60)
```

and 

```py
retry_sleep = step_retry.get('retriesTimeout',{}).get('sleep', 30)
```

Reply:
updated!